packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -23,65 +34,87 @@
+              <span v-text="$gettext('Log in')" />
+            </oc-button>
+          </li>
+        </template>
+        <template v-else>
+          <li>
+            <oc-button
+              id="oc-topbar-account-manage"
+              type="router-link"
+              :to="{ path: '/account' }"
+              appearance="raw"
+            >
+              <avatar-image :width="32" :userid="userId" :user-name="user.displayname" />
+              <span class="profile-info-wrapper" :class="{ 'oc-py-xs': !user.email }">
+                <span class="oc-display-block" v-text="user.displayname" />
+                <span v-if="user.email" class="oc-text-small" v-text="user.email" />
+              </span>
+            </oc-button>
+          </li>
+          <li v-for="(app, index) in applicationsList" :key="`user-menu-${index}`">
+            <oc-button
+              v-if="app.url"
+              type="a"
+              appearance="raw"
+              :target="app.target"
+              :href="app.url"
+            >
+              <oc-icon :name="app.icon" class="oc-p-xs" />
+              <span v-text="$gettext(app.title)" />
+            </oc-button>
+            <oc-button v-else type="router-link" appearance="raw" :to="{ path: app.path }">
+              <oc-icon :name="app.icon" class="oc-p-xs" />
+              <span v-text="$gettext(app.title)" />
+            </oc-button>
+          </li>
+          <li>
+            <oc-button id="oc-topbar-account-logout" appearance="raw" @click="logout">
+              <oc-icon name="logout-box-r" fill-type="line" class="oc-p-xs" />
+              <span v-text="$gettext('Log out')" />
+            </oc-button>
+          </li>
+          <li v-if="quotaEnabled" class="storage-wrapper oc-pl-s">
+            <oc-icon name="cloud" fill-type="line" class="oc-p-xs" />
+            <div class="storage-wrapper-text">
+              <p class="oc-my-rm">
+                <span class="oc-display-block" v-text="personalStorageLabel" />
+                <span class="oc-text-small" v-text="personalStorageDetailsLabel" />
+              </p>
+              <oc-progress
+                v-if="limitedPersonalStorage"
+                :value="quotaUsagePercent"
+                :max="100"
+                size="small"
+                :variation="quotaProgressVariant"
+              />
+            </div>
+          </li>
+        </template>
       </oc-list>
     </oc-drop>
   </nav>
 </template>
 
 <script lang="ts">
-import { defineComponent, PropType, ComponentPublicInstance } from 'vue'
+import { defineComponent, PropType, ComponentPublicInstance, computed } from 'vue'
 import { mapGetters, mapState } from 'vuex'
 import filesize from 'filesize'
 import isNil from 'lodash-es/isNil'
 import { authService } from '../../services/auth'
 import { useCapabilitySpacesEnabled } from 'web-pkg/src/composables'
 import { OcDrop } from 'design-system/src/components'
+import { useRoute } from 'vue-router'
```

Comment:
Usually we import `useRoute` from `web-pkg/src/composables`. I believe it doesn't make a difference in the runtime, but unit tests might fail when imported from `vue-router` because it can't be mocked the way we usually mock stuff.